lib/cjs/generated/browserSessionReplay.d.ts

```diff
@@ -357,8 +357,29 @@ export interface DocumentNode {
      * The type of this Node.
      */
     readonly type: 0;
+    /**
+     * Stylesheet added dynamically
+     */
+    readonly adoptedStyleSheets?: StyleSheet[];
     childNodes: SerializedNodeWithId[];
 }
+/**
+ * Browser-specific. Schema of a StyleSheet
+ */
+export interface StyleSheet {
+    /**
+     * CSS rules applied (rule.cssText)
+     */
+    rules: string[];
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏‏for inline stylesheets, we serialize all rules as a big `_cssText` string attribute. Any reason to use an array here? I'd be fine with it, but bear in mind that some rules can have child rules, so we could still have multiple rules in a single "rule" entry.

Reply:
Indeed, it make sense to reuse `getCssRulesString` in the SDK

Reply:
we chatted, we'll keep the array of string to avoid using `replaceSync` that has a not so good support in the browser or using a hack with `insertRule`